build.sbt

```diff
@@ -1867,6 +1867,7 @@ lazy val deploymentManagerApi = (project in file("designer/deployment-manager-ap
 lazy val designer = (project in file("designer/server"))
   .configs(SlowTests)
   .enablePlugins(GenerateDesignerOpenApiPlugin)
+  .enablePlugins(GenerateScenarioActivityOpenApiPlugin)
```

Comment:
Why do we need a separate plugin for this API?

Reply:
removed separate plugin, those scenario activities api is now included in the main Swagger, including stream-based attachment endpoints